examples/tabpfn_classification.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+import os.path as osp
+
+import numpy as np
+import torch
+from tabpfn import TabPFNClassifier
+# Please run `pip install tabpfn` to install the package
+from tqdm import tqdm
+
+from torch_frame.data import DataLoader
+from torch_frame.datasets import (
+    ForestCoverType,
+    KDDCensusIncome,
+    Mushroom,
+    Titanic,
+)
+
+parser = argparse.ArgumentParser(
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument(
+    '--dataset', type=str, default="Titanic",
+    choices=["Titanic", "Mushroom", "ForestCoverType", "KDDCensusIncome"])
+parser.add_argument('--train_batch_size', type=int, default=4096)
+parser.add_argument('--test_batch_size', type=int, default=128)
+parser.add_argument('--seed', type=int, default=0)
+args = parser.parse_args()
+
+torch.manual_seed(args.seed)
+
+# Prepare datasets
+path = osp.join(osp.dirname(osp.realpath(__file__)), '..', 'data',
+                args.dataset)
+
+if args.dataset == "Titanic":
+    dataset = Titanic(root=path)
+elif args.dataset == "ForestCoverType":
+    dataset = ForestCoverType(root=path)
+elif args.dataset == "KDDCensusIncome":
+    dataset = KDDCensusIncome(root=path)
+else:
+    dataset = Mushroom(root=path)
+
+dataset.materialize()
+assert dataset.task_type.is_classification
+dataset = dataset.shuffle()
+train_dataset, test_dataset = dataset[:0.9], dataset[0.9:]
+train_tensor_frame = train_dataset.tensor_frame
+test_tensor_frame = test_dataset.tensor_frame
+train_loader = DataLoader(
+    train_tensor_frame,
+    batch_size=args.train_batch_size,
+    shuffle=True,
+)
+X_train = []
+train_data = next(iter(train_loader))
```

Comment:
why only sample random minibatch this way?

Reply:
Got it, you probably don't need data loader. Just use index select over the original tensor frame.